services.py

```diff
@@ -22,6 +22,13 @@ def __init__(self, port: int, should_log: bool, frequency: float, decimation: Op
   # service: (should_log, frequency, qlog decimation (optional))
   # note: the "EncodeIdx" packets will still be in the log
   "sensorEvents": (True, 100., 100),
+  "gyroscope": (True, 100., 100),
```

Comment:
aren’t some of these 104Hz?

Reply:
yes, gyro and accel are 104Hz, just thought as we in general aim for 100Hz, changed